### DISCUSSION TITLE
Add authentication lock

### DISCUSSION THREAD

**Discussion by galbeinx** [community/community#3842] [Mobile]
Hey
Is it possible to add this feature? As you opening the app it will require a fingerprint or passcode

**Comment by nplasterer** [comment]
Thanks for the feedback. We added your feedback to our internal tracking issue and hope to have something for this soon.

**Reply by galbeinx** [reply]
Great hope to use it soon!

**Reply by jpsanabria90** [reply]
Is this happening?

**Comment by jonatasrs** [comment]
> Thanks for the feedback. We added your feedback to our internal tracking issue and hope to have something for this soon.

I hope so too.  Security is important.  It could be selectable in the options.

**Comment by aconsuegra** [comment]
This feature is now available on GitHub Mobile! Check the announcement in https://github.blog/changelog/2024-08-06-app-lock-via-face-id-or-biometrics/, and let us know if you have any feedback. Thanks for your patience!

**Reply by ttodua** [reply]
took 3 years to consider this crucial feature.
